pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.black]
+line-length = 119
+target-version = ['py37', 'py38', 'py39']
+
+[tool.isort]
+known_first_party = "autogluon"
+known_third_party = [
+    "numpy",
+    "pandas",
+    "psutil",
+    "pytest",
+    "Pillow",
+    "boto3"
+]
+line_length = 119
+profile = "black"
```

Comment:
nit: extra empty line at the end

Reply:
oops. Just merged it in lol. Will add it in follow-up PRs